pkg/keyspace/tso_keyspace_group.go

```diff
@@ -286,33 +290,70 @@ func (m *GroupManager) GetKeyspaceGroupByID(id uint32) (*endpoint.KeyspaceGroup,
 // DeleteKeyspaceGroupByID deletes the keyspace group by ID.
 func (m *GroupManager) DeleteKeyspaceGroupByID(id uint32) (*endpoint.KeyspaceGroup, error) {
```

Comment:
DeleteKeyspaceGroupByID() can't work alone without timeline merge logic to avoid time fallback, correct?